src/ppx_deriving.mli

```diff
@@ -72,35 +72,39 @@ val string_of_core_type : Parsetree.core_type -> string
     The [~name] argument is used in error messages and should receive
     the name of the deriving plugin, e.g. ["show"]. *)
 module Arg : sig
+  (** Convert raw expression into an argument. Returns [Result.Error "error"]
+      if conversion fails *)
```

Comment:
`A type of conversion functions.` etc, please see above how it's written for the type `deriver`.